code/modules/medical/genetics/bioEffects/hemochromia.dm

```diff
@@ -0,0 +1,442 @@
+/datum/bioEffect/hemochromia
+	name = "Hemochromia Type-U"
+	desc = "A volatile mutation with 12 known stable alternatives. Will quickly break down into one of them if the subject exits the scanner."
+	id = "hemochromia_unknown"
+	probability = 111
+	effectType = EFFECT_TYPE_MUTANTRACE
+	msgGain = "You feel like you're out of breath."
+	msgLose = "You feel more stable."
+	reclaim_fail = 10
+	lockProb = 1
+	lockedGaps = 2
+	lockedDiff = 2
+	lockedChars = list("A","T","C","G")
+	lockedTries = 12
+	stability_loss = 5
+	icon_state  = "hemochromia_unknown"
+	var/typeRange = 0
+	var/duplicateCheck = 0
+	var/blood_color_R
+	var/blood_color_G
+	var/blood_color_B
+
+
+
+	OnLife()
+		if(ishuman(owner))
+			var/mob/living/carbon/human/H = owner
+			if(prob(12))
+				if(!istype(H.loc, /obj/machinery/genetics_scanner))
```

Comment:
This feels hacky, why is it there?

Reply:
It is intentional. The GeneTek scanners prevent the Type-U from degrading, so Geneticists can yoink the mutation without being rushed by time. Once the subject steps out o the scanner, though, they'll soon have the Type-U degrade.